utils/createDiffText.js

```diff
@@ -0,0 +1,100 @@
+const createDiffText = diffTextObject => {
+  const propertyParser = propertyString => {
+    return propertyString.replaceAll(/\.\d/g, "");
+  };
+  const differenceTexts = {};
+
+  for (const property in diffTextObject) {
+    const value = diffTextObject[property];
+
+    if (property.includes("absoluteBoundingBox")) {
+      if (property.includes(".x") || property.includes(".y")) {
+        if (differenceTexts.Position) {
+          differenceTexts.Position = {
+            ...differenceTexts.Position,
+            [propertyParser(property)]: value,
+          };
+        } else {
+          differenceTexts.Position = {
+            [propertyParser(property)]: value,
+          };
+        }
+      } else if (property.includes(".width") || property.includes(".height")) {
+        if (differenceTexts.Size) {
+          differenceTexts.Size = {
+            ...differenceTexts.Size,
+            [propertyParser(property)]: value,
+          };
+        } else {
+          differenceTexts.Size = {
+            [propertyParser(property)]: value,
+          };
+        }
+      }
+    } else if (property.includes("characters") || property.includes("font")) {
+      if (differenceTexts.Text) {
+        differenceTexts.Text = {
+          ...differenceTexts.Text,
+          [propertyParser(property)]: value,
+        };
+      } else {
+        differenceTexts.Text = {
+          [propertyParser(property)]: value,
+        };
+      }
+    } else if (property.includes("Color") || property.includes("background")) {
+      if (differenceTexts.Color) {
+        differenceTexts.Color = {
+          ...differenceTexts.Color,
+          [propertyParser(property)]: value,
+        };
+      } else {
+        differenceTexts.Color = {
+          [propertyParser(property)]: value,
+        };
+      }
+    } else if (differenceTexts.Etc) {
+      differenceTexts.Etc = {
+        ...differenceTexts.Etc,
+        [propertyParser(property)]: value,
+      };
+    } else {
+      differenceTexts.Etc = {
+        [propertyParser(property)]: value,
+      };
+    }
+  }
+
+  let differenceResult = "";
+  for (const type in differenceTexts) {
```

Comment:
P1: 해당 부분에 개행을 해주시는 것이 좋을 것 같습니다

```suggestion
  let differenceResult = "";
  
  for (const type in differenceTexts) {
```

Reply:
변수의 할당과 함수의 선언사이에는 개행이 들어가야하는데
기본적인 컨벤션을 놓쳤네요.. 감사합니다 👍